core/server/data/schema/default-settings.json

```diff
@@ -82,7 +82,7 @@
             "defaultValue": "[{\"url\":\"\"}]"
         },
         "unsplash": {
-            "defaultValue": "[{\"applicationId\":\"\", \"isActive\": true}]"
+            "defaultValue": "{\"applicationId\":\"\", \"isActive\": false}"
```

Comment:
client extensions shouldn't be involved here, it's either config as an override or via settings